woodwork/tests/accessor/test_statistics.py

```diff
@@ -2087,7 +2088,10 @@ def test_get_low_high_bound_warnings():
 
 def test_get_medcouple(outliers_df_pandas, skewed_outliers_df_pandas):
     has_outliers_series = outliers_df_pandas["has_outliers"]
-    has_outliers_series = has_outliers_series.append(pd.Series([39]), ignore_index=True)
+    has_outliers_series = pd.concat(
```

Comment:
Is there a reason we don't concat with `concat_dataframe_or_series` here as well?

Reply:
I used `pd.concat` because it looks like the fixtures are always pandas. Would it be better to generalize this into `concat_dataframe_or_series`?

Reply:
Probably doesn't matter too much, but might be nice to be consistent in how we handle this unless there is a reason to do it differently.

Reply:
EDIT: Actually, this was because this concat call makes use of the `ignore_index` parameter, which we don't support in `concat_dataframe_or_series`